quickstarts/microsoft.maps/maps-create/azuredeploy.json

```diff
@@ -2,43 +2,55 @@
     "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
-        "accountName":{
+        "accountName": {
             "type": "string",
             "defaultValue": "[uniqueString(resourceGroup().id)]",
             "metadata": {
                 "description": "The name for your Azure Maps account. This value must be globally unique."
             }
         }, 
+        "location": {
+            "type": "string",
+            "allowedValues": [
+                "westeurope",
+                "eastus",
+                "westus2",
+                "northeurope",
+                "westcentralus",
+                "usgovvirginia",
+                "usgovarizona"
+            ],
+            "metadata": {
+                "description": "Specifies the location for all the resources."
+            }
+        },
         "pricingTier": { 
             "type": "string", 
             "allowedValues":[ 
-                "S0", 
-                "S1", 
                 "G2"
             ], 
             "defaultValue": "G2",
             "metadata": { 
-                "description": "The pricing tier for the account. Use S0 for small-scale development. Use S1 or G2 for large-scale applications." 
+                "description": "The pricing tier SKU for the account." 
             } 
         }, 
         "kind": { 
             "type": "string", 
             "allowedValues":[ 
-                "Gen1", 
                 "Gen2" 
             ], 
             "defaultValue": "Gen2", 
             "metadata": { 
-                "description": "The pricing tier for the account. Use Gen1 for small-scale development. Use Gen2 for large-scale applications." 
+                "description": "The pricing tier for the account." 
             } 
         } 
     }, 
     "resources": [
         {
             "name": "[parameters('accountName')]",
             "type": "Microsoft.Maps/accounts",
-            "apiVersion": "2021-02-01",
-            "location": "global",
+            "apiVersion": "2021-12-01-preview",
```

Comment:
Shouldn't the `apiVersion` be **2023-06-01**? See [here](https://learn.microsoft.com/rest/api/maps-management/accounts)

Reply:
Thanks for pointing this out. Updtaed.